### PR TITLE
PR CL-134 Create window Google TFA

### PR DESCRIPTION
**Pull request by sinerger**

### CODE REVIEW

**Review by Aryewar — COMMENTED** [review]
6 commits

File: src/components/Modals/GoogleTFAModalWindow/GoogleTFAModalWindowLogic.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    connectTFAToLead(object.Input_Pin, this.state.leadID).then((response) => {
+      this.setState({ isConnectTFA: response.data });
+
+      if(this.state.isConnectTFA){
+        this.setState({ isOpen: false });
+      }
+    });
+
+
```
Comment (Aryewar) [inline src/components/Modals/GoogleTFAModalWindow/GoogleTFAModalWindowLogic.jsx]: Отступы

**Review by Aryewar — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ function App() {
         <Route path = "/register">
           <RegistrationForm/>          
         </Route>
```
Comment (MykhailoKnysh) [inline src/App.js:21]: You have to use "path...." in <Route>

File: src/components/Modals/GoogleTFAModalWindow/GoogleTFAModalWindowLogic.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
+import React, { Component, Fragment } from "react";
+import Modal from "../BaseModalWindow/Modal";
+import QRCode from "./QRCodeForLead";
+import { connectTFAToLead } from "../../../requests";
+import "./GoogleTFAModal.css";
+
+class GoogleTFAModalWindowLogic extends Component {
+  state = {
+    leadID: "1a0359d8-6618-4f94-be03-a9589bc1b43d",
```
Comment (MykhailoKnysh) [inline src/components/Modals/GoogleTFAModalWindow/GoogleTFAModalWindowLogic.jsx:9]: Don`t forget to change leadID to parameter

File: src/components/account/account.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        
+      }
+
+      render() {
+       return (
+           <div></div>
+       );
+      }
+
+}
```
Comment (MykhailoKnysh) [inline src/components/account/account.js:20]: Shift Alt F

**Pull request merged**